.travis.yml

```diff
@@ -31,7 +31,7 @@ before_script:
 - "./tests/before_script.sh"
 script:
   - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-install -ncc
-  - php scripts/mqtt.php --tests > /dev/null 2>&1 &
+  - #php scripts/mqtt.php --tests > /dev/null 2>&1 &
```

Comment:
Still not used yet. I think I'll use the daemon with DI (see toe PR in experimentation)